bin/varnishd/mgt/mgt_vcl.c

```diff
@@ -33,7 +33,10 @@
 
 #include "config.h"
 
+#include <errno.h>
```

Comment:
From what @dridi told me the policy is that we try to keep inclusions in .c files as much as possible

Reply:
So if it was a policy, I guess we should write it down ;)

As far as my personal preferences go, I have not started using a [language server](https://releases.llvm.org/8.0.0/tools/clang/tools/extra/docs/clangd/Installation.html), which only works well with all needed includes present where used. So for this practical aspect, I would prefer a policy in to that effect, but otherwise I really don't care much.

Reply:
I do use an LSP (clangd) and it does fine with this way of including files